src/binaryen-c.cpp

```diff
@@ -3399,59 +3365,82 @@ BinaryenTableRef BinaryenGetTableByIndex(BinaryenModuleRef module,
   }
   return tables[index].get();
 }
-
-int BinaryenIsFunctionTableImported(BinaryenModuleRef module) {
-  if (((Module*)module)->tables.size() > 0) {
-    return ((Module*)module)->tables[0]->imported();
+BinaryenElemSegmentRef
+BinaryenAddActiveElemSegment(BinaryenModuleRef module,
```

Comment:
What do you think of spelling out "Elem" as "Element" here and in similar places? Would feel a bit more consistent to me, since we also do not shorten "Function" to "Func" or similar in function names. Not feeling strongly, though.

Reply:
Hmm so `BinaryenAddActiveElementSegment` or `BinaryenAddActiveElement`? Making it consistent would require changing the name across the Binaryen IR as well. 🤔 I don't feel strongly either, it just makes statements longer, specially inside Binaryen.

Reply:
By the way, for this specific API, I can also provide a simpler version that can be used for both active and passive segments. So if null is provided for table name, I'll make it a passive segment otherwise it'll be active. My initial idea was that making it explicit might help with readability.

Reply:
I would support changing "Elem" to "Element" everywhere, but I also don't feel strongly about it as long we use either one of those consistently.

Reply:
FYI, I changed Elem to Element everywhere. Doesn't look bad.